recipes/datalad-container/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "datalad-container" %}
+{% set pypiname = "datalad_container" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz
+  sha256: 3e9c87e1aa712455863c8755fc480413297ddd820086312753123042fed8c139
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<32]
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - pypandoc
+  run:
+    - python
```

If (and only if) it is made noarch:
```suggestion
    - python >=3.2
```